### PR TITLE
Enabled integration step in CICD

### PR DESCRIPTION
**Pull request by vijayg10**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -143,43 +143,76 @@ jobs:
...
-      - checkout
       - run:
-          name: Set up NVM
+          name: Change the node version
           command: |
-            echo ${NVM_DIR}
             [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-            nvm install v14.17.6
-            nvm alias default v14.17.6
+            nvm install v12.22.1
```
Comment (kleyow) [inline .circleci/config.yml]: Something wrong with the LTS version of node?

File: .circleci/config.yml [hunk 2]
```diff
@@ -243,19 +276,14 @@ jobs:
...
-          name: Setup Docker env variables
-          command: |
-            echo 'export DOCKER_ORG=`cat package.json | jq -r .name | sed s/@// | sed s/\\\\/.*//`' >> $BASH_ENV
-            echo 'export CIRCLE_PROJECT_REPONAME=`cat package.json | jq -r .name | sed s/@// | sed s/\.*\\\\///`' >> $BASH_ENV
       - run:
           name: Build Docker local image
           command: |
             echo "Building Docker image: local"
-            docker build -t $DOCKER_ORG/$CIRCLE_PROJECT_REPONAME:local .
+            docker build -t ${DOCKER_ORG:-mojaloop}/$CIRCLE_PROJECT_REPONAME:local .
```
Comment (kleyow) [inline .circleci/config.yml:280]: whats the reason for this?

File: README.md [hunk 3]
```diff
@@ -3,25 +3,68 @@
...
+spec:
+  # must match what it is used in Keto, whatever that is
+  role: RoleIdentifier
+  permissions:
+  - permission_01
+  - permission_02
+  - permission_03
+  - permission_04'
+```
+This sequence digram illustrates how Ort Keto is updated.
```
Comment (kleyow) [inline README.md]: nit
```suggestion
This sequence digram illustrates how Ory Keto is updated.
```

File: .circleci/config.yml (hunk 2 above)
Comment (vijayg10) [reply .circleci/config.yml:280]: The DOCKER_ORG variable is organisation level variable in circleCI that doesn't propagate incase of forked PR's. So I just want to pass a default value if that variable is not set.

File: .circleci/config.yml (hunk 1 above)
Comment (vijayg10) [reply .circleci/config.yml]: Not sure why I changed this. Let me try to use the latest version. Thanks for spotting it.

**Review by kleyow — APPROVED** [review]

**Pull request merged**